runelite-client/src/main/java/net/runelite/client/plugins/hiscore/HiscoreConfig.java

```diff
@@ -74,4 +74,15 @@ default boolean autocomplete()
 	{
 		return true;
 	}
+
+	@ConfigItem(
+			position = 5,
+			keyName = "bountylookup",
+			name = "Bountylookup",
+			description = "Automatically lookup the stats of your bounty hunter target"
+	)
+	default boolean bountylookup()
+	{
+		return true;
```

Comment:
default this to false

Reply:
Actually, I disagree with this one. Since most users won't interact with bounty hunter to begin with, I feel it would be better for PvP qol to enable this for those users who do play bounty hunter.

Reply:
Well this lookups on the chat message. And you can be on bounty hunter world without caring about their targets but they still get them assigned